src/report/generate_trustymail_report.py

```diff
@@ -251,9 +251,9 @@ def add_weak_crypto_data_to_domain(domain_doc, sslyze_data_all_domains):
         all_domains_cursor.close()
 
         # Get count of second-level domains an agency owns
-        self.__base_domain_count = self.__db.trustymail.find(
+        self.__base_domain_count = self.__db.trustymail.count_documents(
             {"latest": True, "agency.name": agency, "is_base_domain": True}
-        ).count()
+        )
```

Comment:
Nice performance improvement! 👍

Reply:
I still don't understand how this suddenly became an issue after August 7, but I've wasted enough time trying to figure that out. :disappointed:

Reply:
Yeah, that's strange, but maybe that large influx of subdomains that we discussed pushed us over the edge somehow.  Anyway, thanks for finding something that works here.